plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaBuilderTest.kt

```diff
@@ -61,7 +61,7 @@ class JavaLambdaBuilderTest {
     }
 
     @Test
-    fun mavenRootPomHandlerBaseDirIsCorrect() = runTest {
+    fun mavenRootPomHandlerBaseDirIsCorrect(): Unit = runBlocking {
```

Comment:
it would also be good to add a comment in the source code explaining why we're using `runBlocking`
```suggestion
    fun mavenRootPomHandlerBaseDirIsCorrect() = runBlocking {
```

Reply:
sure Definitely